Mathlib/RingTheory/PicardGroup.lean

```diff
@@ -4,15 +4,11 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Junyan Xu
 -/
 import Mathlib.Algebra.Category.ModuleCat.Monoidal.Symmetric
-import Mathlib.Algebra.Module.FinitePresentation
-import Mathlib.Algebra.Module.LocalizedModule.Submodule
 import Mathlib.CategoryTheory.Monoidal.Skeleton
 import Mathlib.LinearAlgebra.Contraction
-import Mathlib.LinearAlgebra.TensorProduct.Finiteness
-import Mathlib.LinearAlgebra.TensorProduct.RightExactness
 import Mathlib.LinearAlgebra.TensorProduct.Submodule
-import Mathlib.RingTheory.Flat.Localization
-import Mathlib.RingTheory.Localization.BaseChange
+import Mathlib.RingTheory.ClassGroup
```

Comment:
Only this import is relatively heavy and is triggering large-import. If necessary, I can add another file PicardGroup/ClassGroup and rename this file to PicardGroup/Basic. Or just ignore the large-import since this is a leaf file ...